docs/en/ingest-management/elastic-agent/install-elastic-agent.asciidoc

```diff
@@ -1,31 +1,65 @@
 [[elastic-agent-installation]]
 [role="xpack"]
-= Install {agent} and enroll it in {fleet}
+= Install {agent}s
 
 beta[]
 
-Install an {agent} on each host that you want to monitor. These steps assume
-that you're running a fresh installation. If {agent} is already running on your
-system and you want to upgrade to a new version, see <<upgrade-elastic-agent>>.
+To collect and ship data to the {stack}, install an {agent} on each host you
+want to monitor. These steps assume that you're running a fresh installation. If
+{agent} is already running on your system and you want to upgrade to a new
+version, see <<upgrade-elastic-agent>>.
```

Comment:
I love this linking so much - thank you for the effort to improve the navigability